addons/web_hierarchy/static/src/hierarchy_model.js

```diff
@@ -116,6 +116,15 @@ export class HierarchyNode {
         return this.model.childFieldName || this.model.defaultChildFieldName;
     }
 
+    /**
+     * Get the display name of the node
+     *
+     * @returns {String}
+     */
+    get displayName() {
+        return this.data.display_name || this.data.name;
```

Comment:
Isn't "data.display_name" always provided inside the retrieved data? Would allow to simplify a bit, not sure we need a hook for this.

Reply:
The idea that ADSC suggested is that we have this hook instead of a field name in the templates so that it can be overridden by anything more easily, and the template does not need to be modified (less xpath ?)

But in current cases `display_name` should be enough so maybe it's not necessary right now

Reply:
I agree we could just use the display_name field except if somewhere we need to override that.